chainerio/containers/zip.py

```diff
@@ -41,15 +29,50 @@ def _check_zip_file_name(self, base):
 
     def _open_zip_file(self, mode='r'):
         mode = mode.replace("b", "")
+        zip_file = self.base_handler.open(self.base, "rb")
+
         if self.zip_file_obj is None:
-            zip_file = self.base_handler.open(self.base, "rb")
+            if isinstance(self.base_handler, ZipContainer) \
+                    and sys.version_info < (3, 7, ):
```

Comment:
My opinion was not like this; to reopen B.zip and replace the file object right before opening A in B.zip in C.zip, not when opening B.zip in C.zip.

Reply:
Correct me if I am thinking in a wrong way.
>to reopen B.zip and replace the file object right before opening A in B.zip in C.zip, not when opening B.zip in C.zip.
I don't think this is possible because zip file module doesn't do lazy open, hence they check the seekable upon creating the zipfile object. If the file object passed to the zipfile.ZipFile is not seekable, an exception is thrown at that point.

Reply:
As we discussed offline, we will remove this issue to another PR